### PR TITLE
Update AsciiDoc preview with more examples

### PR DESCRIPTION
**Pull request by eric-schneider**

### CODE REVIEW

**Comment by eric-schneider** [comment]
@mlr Looks like I pulled in package-lock by accident and have to resolve some merge conflicts.

**Comment by mlr** [comment]
> @mlr Looks like I pulled in package-lock by accident and have to resolve some merge conflicts.

We should remove package-lock from gitignore to ensure everyone has the same dependencies. Since your PR didn’t alter any dependencies it should be safe to use the contents currently on main

File: README.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ ifdef::env-github[]
 endif::[]
 // Project URLs:
 :url-project: https://github.com/riptano/docs-ui
-:url-preview: https://riptano.github.io/docs-ui/
+:url-preview: https://fantastic-adventure-593lpwy.pages.github.io/main/home/index.html
```
Comment (mlr) [inline README.adoc:21]: I think we should change this to https://riptano.github.io/datastax-docs-site/main
two reasons 1) /home is the old build - need to have GH pages clean that up
2) the URL https://riptano.github.io/datastax-docs-site/main will auto redirect to fantastic-adventure, and if that should change for any reason, we don't need to worry about the README being out of date

Comment (eric-schneider) [reply README.adoc:21]: Great catch!

Comment (eric-schneider) [inline README.adoc:21]:
```suggestion
:url-preview: https://riptano.github.io/datastax-docs-site/main
```

**Review by mlr — APPROVED** [review]

**Pull request merged**